### PR TITLE
Increased space between title and tabs.

### PR DESCRIPTION
**Pull request by vikaschandak**
Changes layout height from 80 dp to 92 dp.

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -3,7 +3,9 @@
   <component name="ProjectModuleManager">
```
Comment (vishwesh3) [inline .idea/modules.xml]: Put this file in `.gitignore`

File: app/src/main/res/layout/app_bar_main.xml [hunk 2]
```diff
@@ -49,7 +49,7 @@
             <android.support.v7.widget.Toolbar
                 android:id="@+id/toolbar"
                 android:layout_width="match_parent"
-                android:layout_height="80dp"
+                android:layout_height="92dp"
```
Comment (vishwesh3) [inline app/src/main/res/layout/app_bar_main.xml]: In my opinion this is not the right way :)
try something with `margin`
and use `dimen.xml`

File: .idea/modules.xml (hunk 1 above)
Comment (vishwesh3) [reply .idea/modules.xml]: https://stackoverflow.com/a/4308645/5612089

File: app/src/main/res/layout/app_bar_main.xml (hunk 2 above)
Comment (vishwesh3) [reply app/src/main/res/layout/app_bar_main.xml]: https://stackoverflow.com/questions/29128178/how-to-use-dimens-xml-in-android

**Comment by vishwesh3** [comment]
And also squash your commits into 1
https://vishwesh3.github.io/i-introduction-to-git/
https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Comment by vikaschandak** [comment]
I am relatively new to android so I will have to learn some more things. I am closing this PR and opening the new one with more issues resolved.

**Pull request closed without merging**